### ISSUE TITLE
hello，the MEF-SSIMc is mean to mef-ssim-color？

### ISSUE THREAD

**Issue by potoyeee** [johnmmm/MEF-MS-SSIM_implementation#1]
is it same to the MEF-SSIM-c in “Deep unsupervised learning based on color un-referenced loss functions for multi-exposure image fusion”？

**Comment by johnmmm** [comment]
Yes, this is the implementation of "Multi-Exposure Image Fusion by Optimizing A Structural Similarity Index".
